### PR TITLE
Completed es6-destructuring | Daniel To

### PR DESCRIPTION
**Pull request by theDanielTo**
image

### CODE REVIEW

File: es6-destructuring/main.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  {
+    title: "Dead Dead Demon's De De De De Destruction",
+    author: 'Inio Asano',
+    libraryID: 1233
+  }
+];
+const [book3, book4, book5] = library;
+console.log('book3:', book3);
+console.log('book4:', book4);
+console.log('book5:', book5);
```
Comment (scbowler) [inline es6-destructuring/main.js:42]: Looks like you missed the last step, number 15:

```
15. Log the last book object to the console with the appropriate console label.

```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just looks like you missed the last step

File: es6-destructuring/main.js (hunk 1 above)
Comment (theDanielTo) [reply es6-destructuring/main.js:42]: image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**